src/components/tooltip/Tooltip.tsx

```diff
@@ -73,6 +73,8 @@ export type DefaultTooltipProps = {
     constrainToWindow: boolean;
     /** Forces the tooltip to be disabled irrespecitve of it's shown state. Defaults to `false` */
     isDisabled: boolean;
+    /** A string of the form 'vert-offset horiz-offset' which controls positioning */
+    offset: string;
```

Comment:
looks to me offset can be part of TooltipCustomPosition. what do you think?

Reply:
I looked into this, and don't think I agree for a few reasons

1) The offset is different from the position prop 
2) if we want to pass an offset, we need to pass a custom position instead of the string
3) it makes typescript typing more difficult as there are 2 types for the tooltip position